paths/electron/starting-with-electron/02-electron-boilerplate.md

```diff
@@ -20,7 +20,7 @@ main-content: |
   ![gif of following the directions below](<% SITEURL %><% BASEURL %>/images/gifs/electron/electron1-createapp.gif)
 
   1. Find the terminal and navigate to your desired project location, for example: `cd ~/` will navigate to your home directory.
-  1. Install electron-forge globally: `npm install -g electron-forge`
+  1. Install electron-forge globally: `npm install electron-forge`
```

Comment:
Hrm. I don't think that's going to work here. Ideally, you'd say something like this:

> 1. Initialize a new project: `npx electron-forge init electron-app` or `yarn create electron-app electron-app`
> 1. Change into that app's directory: `cd electron-app`

Unfortunately, I believe that in another file you recommend NPM 3 _(which I endorse because of weird behavior in more recent versions of NPM - I'm waiting to see if 5.4.0 resolves them)_, so for now it might make more sense to leave it as it is.